examples/custom-server-express/server.js

```diff
@@ -9,15 +8,6 @@ app.prepare()
 .then(() => {
   const server = express()
 
-  server.use((req, res, next) => {
-    const accept = accepts(req)
-    const type = accept.type(['json', 'html'])
-    if (type === 'json') {
-      return handle(req, res)
-    }
-    next()
-  })
```

Comment:
I was rather thinking how to make handling JSON easier, for example, we can support a middleware-like API.

```js
server.use(app.handleJSON())
```

If server is not `express`:

```js
handle(req, res) {
  app.handleJSON(req, res, (err) => {
    // called only if Accept header is not JSON.
  })
}
```